torch/utils/data/_utils/worker.py

```diff
@@ -327,3 +341,6 @@ def _worker_loop(dataset_kind, dataset, index_queue, data_queue, done_event,
     if done_event.is_set():
         data_queue.cancel_join_thread()
         data_queue.close()
+
+def _generate_seed(generator):
+    return torch.empty((), dtype=torch.int64).random_(generator=generator).item()
```

Comment:
nit: pass `device='cpu'` here so that torch.device doesn't stupidly move this to CUDA lol